pdp/pdp.go

```diff
@@ -47,28 +42,26 @@ func CanAccess(accRec AccessRequest) AccessResponse {
 }
 
 /*
-GetPolicies loads a list of policies from the policy directory and return an array of AccessPolicy.
+Init loads a list of policies from a yaml file.
 */
-func GetPolicies() []AccessPolicy {
-	f, err := policies.ReadFile("policy/policy_example.yaml")
+func Init(policyFile string) error {
```

Comment:
I think `Init` is a bad name even tough I was the one suggesting it :| It is to similar to `init` which is a special method in go.

Maybe call it `SetPolicies`.